inspectit-oce-documentation/src/docs/asciidoc/_includes/metrics/metrics-exporters.adoc

```diff
@@ -0,0 +1,43 @@
+=== Metrics Exporters
+
+Metrics exporters are responsible for passing the collected metrics to metric storage.
+They can implement a push approach where metrics are sent to a collector or a pull approach where metrics are scraped by an external system.
+
+If an exporter supports run-time updates it means that it can be enabled/disabled during the run-time or that any property related to the exporter can be changed.
+This way you can, for example, change the endpoint where exporter pushes the metrics without a need to restart the application.
+In order to use run-time updates, you must enable one of the <<Externalized Configuration Methods,externalized configuration methods>> that support dynamic updates.
+
+inspectIT OCE current supports following OpenCensus metrics exporters:
+
+[cols="1,1,1,1",options="header"]
+|===
+|Exporter |Supports run-time updates| Push / Pull |Enabled by default
+|<<Prometheus Exporter>>
+|Yes
+|Pull
+|Yes
+|===
+
+==== Prometheus Exporter
+
+Prometheus exporter exposes the metrics in Prometheus format and is the default metrics exporter set up by inspectIT OCE.
+It requires that a Prometheus HTTP server is started in parallel with your application.
+The server is by default started by the inspectIT OCE agent on port `8888` and metrics can then be accessed by visiting http://localhost:8888/metrics.
+
+[cols="3,1,3",options="header"]
+.Properties related to the Prometheus exporter
+|===
+|Property |Default| Description
+|```inspectit.exporters.metrics.prometheus.enabled```
+|`true`
+|If true, the inspectIT OCE agent will try to start the Prometheus stats exporter and Prometheus HTTP server.
```

Comment:
would call it metrics exporter and not stats exporter here. People who don't know OpenCensus might be confused by the term "stats".